docs/json-map.md

```diff
@@ -0,0 +1,72 @@
+# Overview
+JSON Map is a JSON-based format which describes part of the structure of another JSON document. Maps inform clients of the sizes of parts of the JSON and to allow constant time construction of subsets of the document.
```

Comment:
>and to allow [](start = 150, length = 12)

Either "and allow" or "to allow". I like "to allow" better because that's the _reason_ for doing it, not just another, independent _consequence_ of doing it.

Reply:
fixed